composer.json

```
@@ -40,6 +40,7 @@
     },
     "require-dev": {
         "facile-it/paraunit": "^1.3 || ^2.0",
+        "infection/infection": "^0.27",
```

```suggestion
        "infection/infection": "~0.27",
```

due to semver, 0.28 can break BC breaks

---

Again, [it's not needed](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7874#issuecomment-1991641902).

---

TIL, nice